### DISCUSSION TITLE
Video attachment in an Issue comment can be previewed, but when the comment is posted the video only shows a link to download

### DISCUSSION THREAD

**Discussion by wangshuNex** [community/community#63338] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Lemme try to post something here as an example.
Sometimes in an Issue comment, when I post a QuickTime screen recording MOV file, it  can be previewed in the "Preview" section, but only shows a link when I post the comment.

https://github.com/community/community/assets/104790504/90d3ed8a-7964-4182-966d-30e7ba4ec6d4

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by wangshuNex** [comment]
Trying in the comment


https://github.com/community/community/assets/104790504/09d852bf-e335-40ac-871e-1576aa25a622

**Comment by wangshuNex** [comment]
The above on works, but the other video I posted in my workspace only shows a link instead of the video per se.

**Comment by dmendonez-mdsol** [comment]
@wangshuNex - The resolution is that the video file link needs to have a blank line above and below it.

**Reply by wangshuNex** [reply]
Thanks! Will try the next time 👍

**Reply by smithl** [reply]
this!
